sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -74,50 +62,52 @@ export class AttestationResponse<T> {
 }
 
 // @public
-export interface AttestationResult {
-    cnf?: any;
-    deprecatedEnclaveHeldData?: Uint8Array;
-    deprecatedEnclaveHeldData2?: Uint8Array;
-    deprecatedIsDebuggable?: boolean;
-    deprecatedMrEnclave?: string;
-    deprecatedMrSigner?: string;
-    deprecatedPolicyHash?: Uint8Array;
-    deprecatedPolicySigner?: JsonWebKey;
-    deprecatedProductId?: number;
-    deprecatedRpData?: string;
-    deprecatedSgxCollateral?: any;
-    deprecatedSvn?: number;
-    deprecatedTee?: string;
-    deprecatedVersion?: string;
-    enclaveHeldData?: Uint8Array;
-    exp?: number;
-    iat?: number;
-    inittimeClaims?: any;
-    isDebuggable?: boolean;
-    iss?: string;
-    jti?: string;
-    mrEnclave?: string;
-    mrSigner?: string;
-    nbf?: number;
-    nonce?: string;
-    policyClaims?: any;
-    policyHash?: Uint8Array;
-    policySigner?: JsonWebKey;
-    productId?: number;
-    runtimeClaims?: any;
-    sgxCollateral?: any;
-    svn?: number;
-    verifierType?: string;
-    version?: string;
-}
+export class AttestationResult {
```

Comment:
we already talked about this offline and I think you'll update those models to be interfaces, right?

Reply:
AttestationSigningKey? Yes. AttestationResult? No, it's a returned-only type and should be a class.

Reply:
It was an interface, what is wrong with keeping it this way? I do not see any interesting member methods there.